mujoco_warp/_src/collision_convex.py

```diff
@@ -965,9 +965,10 @@ def gjk_epa_sparse(
 
 def gjk_narrowphase(m: Model, d: Data):
   if len(_collision_kernels) == 0:
-    for types in _CONVEX_COLLISION_FUNC:
-      t1 = types[0]
-      t2 = types[1]
+    for i in range(m.convex_collision_pair.shape[0]):
+      convex_collision_pair = m.convex_collision_pair[i]
+      t1 = int(convex_collision_pair[0])
+      t2 = int(convex_collision_pair[1])
       _collision_kernels[(t1, t2)] = _gjk_epa_pipeline(
```

Comment:
With this approach do we still need a module level cache?  Why not just do:

```py
for t1, t2 in m.convex_collision_pair:
  kernel = _gjk_epa_pipeline(...
  wp.launch(kernel, ...
```

Reply:
done